File: 프로그래머스/Level1/K번째수/kst_num.py
"""
리스트 슬라이싱과 정렬을 사용하여 문제 해결
"""
def solution(array : list, commands : list) -> list:
    answer = []
    for i,j,k in commands:
        divide_array = array[i-1:j]
        divide_array.sort()
        answer.append(divide_array[k-1])
    return answer

File: 프로그래머스/Level1/K번째수/kst_num.js
function solution(array, commands) {
    var answer = [];
    for (let command of commands){
        const [j,k,l] = command;
        const slice = array.slice(j-1, k).sort((a,b)=>a-b);
        answer.push(slice[l-1]);
    }
    return answer;
}